repos/system_upgrade/el7toel8/actors/inhibitwhenluks/actor.py

```diff
@@ -12,17 +13,19 @@ class InhibitWhenLuks(Actor):
 
     name = 'check_luks_and_inhibit'
     consumes = (StorageInfo,)
-    produces = (Inhibitor,)
+    produces = (Report,)
     tags = (ChecksPhaseTag, IPUWorkflowTag)
 
     def process(self):
         for storage_info in self.consume(StorageInfo):
             for blk in storage_info.lsblk:
                 if blk.tp == 'crypt':
-                    self.produce(Inhibitor(
-                        summary='LUKS encrypted partition detected',
-                        details='Upgrading system with encrypted partitions is not supported',
-                        solutions='If the encrypted partition is not system one and the system '
-                                  'is not depending on it, you can remove/blacklist it from '
-                                  'the system'))
+                    report_with_remediation(
+                        severity='high',
```

Comment:
Does it make sense to set the "severity" when it is `inhibitor`? Just reading that - would be severity anything else than the highest one when we inhibit that?

Reply:
Default severity is currently "medium" and in the framework there is a check if there is inhibitor flag and severity != high, raise an error. But this is not firm, we just got to this after discussion with @shaded-enmity. I am in favour of setting it explicitly to high.